.github/workflows/release/starknet-contracts.yml

```diff
@@ -0,0 +1,56 @@
+name: Starknet Contracts Release
+
+on:
+  push:
+    tags:
+      - "@chainlink/starknet-contracts@[0-9]+.[0-9]+.[0-9]"
+
+jobs:
```

Comment:
We also need to create a GH Release when this tag is pushed via https://github.com/softprops/action-gh-release

This should be the first job we execute, and we should also publish/upload contract artifacts to GH Release.

Reply:
Also check out: https://github.com/smartcontractkit/chainlink-starknet/blob/develop/.github/workflows/release/starknet-relayer.yml